internal/wasm/jit/arch_arm64.go

```diff
@@ -0,0 +1,97 @@
+package jit
+
+import (
+	"fmt"
+	"math"
+
+	asm "github.com/twitchyliquid64/golang-asm"
+	"github.com/twitchyliquid64/golang-asm/obj/arm64"
+
+	wasm "github.com/tetratelabs/wazero/internal/wasm"
+	"github.com/tetratelabs/wazero/internal/wazeroir"
+)
+
+func init() {
```

Comment:
helps clarify when looking at files as not always notice the file name.

```suggestion
// init initializes variables for the arm64 architecture
func init() {
```